config.go

```diff
@@ -181,26 +186,19 @@ func init() {
 			Outputs:     map[string]string{"console": ""},
 		},
 	}
-	ParseConfig()
-}
 
-// ParseConfig parsed default config file.
-// now only support ini, next will support json.
-func ParseConfig() (err error) {
-	if AppConfigPath == "" {
-		// initialize default configurations
-		AppConfigPath = filepath.Join(AppPath, "conf", "app.conf")
-		if !utils.FileExists(AppConfigPath) {
-			AppConfig = &beegoAppConfig{config.NewFakeConfig()}
-			return
-		}
+	AppConfigPath = filepath.Join(AppPath, "conf", "app.conf")
```

Comment:
Yes, not need, but the next release will be a minor version，so i think not changing the Public variable may be better